src/AccessibilityInsights.SharedUx/Controls/EventRecordControl.xaml.cs

```diff
@@ -95,8 +101,12 @@ public static ConfigurationModel AppConfiguration
         public EventRecordControl()
         {
             InitializeComponent();
-
             InitCommandBindings();
+
+            player = new SoundPlayer();
+            Assembly assembly = Assembly.GetExecutingAssembly();
+            startRecordingSoundStream = assembly.GetManifestResourceStream(assembly.GetName().Name + ".Resources.Sound.start_event_recording.wav");
```

Comment:
Should this filename be referred to by a constant or resource string somewhere?

Reply:
I would say yes if we were referencing it in multiple places. In this case, though, it seems simpler to just have the string inline as you've done.